numpy/add_newdocs.py

```diff
@@ -1453,7 +1453,7 @@ def luf(lamdaexpr, *args, **kwargs):
         When True, yield `x`, otherwise yield `y`.
     x, y : array_like, optional
         Values from which to choose. `x` and `y` need to have the same
-        shape as `condition`.
+        shape as `condition`, or be broadcastable to that shape.
```

Comment:
.. which would also cover the aforementioned (1,3), (3,1) and () case as well as the case where all shapes are the same, right?